flash/core/data/data_module.py

```diff
@@ -1152,3 +1153,95 @@ def from_fiftyone(
             num_workers=num_workers,
             **preprocess_kwargs,
         )
+
+    @classmethod
+    def from_labelstudio(
+        cls,
+        export_json: str = None,
+        img_folder: str = None,
```

Comment:
Should be data_folder there or label studio supports only images ?

Reply:
Fixed